inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/TimestampExtractFunction.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sdk.transform.process.function;
+
+import org.apache.inlong.sdk.transform.decode.SourceData;
+import org.apache.inlong.sdk.transform.process.Context;
+import org.apache.inlong.sdk.transform.process.operator.OperatorTools;
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * TimestampExtractFunction
+ * description:
+ * - hour(timestamp)--returns the hour of a day (an integer between 0 and 23) from SQL timestamp timestamp
+ * - minute(timestamp)--returns the minute of an hour (an integer between 0 and 59) from SQL timestamp timestamp
```

Comment:
```suggestion
 * - hour(timestamp)--returns the hour of a day (an integer between 0 and 23) from SQL timestamp
 * - minute(timestamp)--returns the minute of an hour (an integer between 0 and 59) from SQL timestamp
```

Reply:
This description is copy from the Flink document, I think the `SQL timestamp` may be a type, and the following `timestamp` may be the name of the parameter.
![image](https://github.com/user-attachments/assets/766a58c0-c931-472e-be2a-d36b4bf7d1df)

Reply:
fixed.